features/features.go

```diff
@@ -59,6 +59,8 @@ const (
 	MandatoryPOSTAsGET
 	// Use an optimized query for GetOrderForNames.
 	FasterGetOrderForNames
+	// Allow creation of new accounts (on by default)
```

Comment:
Can you also update this comment to emphasize that it's for new ACME v1 registrations and not new ACME v2 account creation?